### PR TITLE
add wordle extension

### PR DESCRIPTION
**Pull request by chrisalxlng**
## Description

This is a new extension that allows users to play Wordle offline in Raycast. It currently supports English and German word sets. It has 3 commands:

- `Play English (US)`: Play Wordle with an English word set
- `Play German (DE)`: Play Wordle with a German word set
- `Show History`: Show summaries of past puzzles

## Screencast

https://user-images.githubusercontent.com/65340138/217403793-c730cbf2-00c4-4dca-abfd-a1147b5c4558.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/wordle/package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "icon": "wordle-icon.png",
+  "author": "chrisalxlng",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "play_en-US",
+      "title": "Play English (US)",
```
Comment (pernielsentikaer) [inline extensions/wordle/package.json:15]:
```suggestion
      "title": "Play English (US)",
      "subtitle": "Wordle",
```

File: extensions/wordle/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      "description": "Play Wordle with an English word set",
+      "mode": "view"
+    },
+    {
+      "name": "play_de-DE",
+      "title": "Play German (DE)",
```
Comment (pernielsentikaer) [inline extensions/wordle/package.json:21]:
```suggestion
      "title": "Play German (DE)",
      "subtitle": "Wordle",
```

File: extensions/wordle/package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      "description": "Play Wordle with a German word set",
+      "mode": "view"
+    },
+    {
+      "name": "show_history",
+      "title": "Show History",
```
Comment (pernielsentikaer) [inline extensions/wordle/package.json:27]:
```suggestion
      "title": "Show History",
      "subtitle": "Wordle",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* It seems like it's case-sensitive, which might confuse everyone 😄

https://user-images.githubusercontent.com/38033107/217859286-43a8bffd-01b4-407c-9db7-fd22388576c5.mp4

* Maybe also add a Delete Summary action

Raycast-1tkwV3G4@2x

- Is it possible to change the action so it matches the one from the dropdown if de is selected?

Raycast-PmfA5VVG@2x


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by chrisalxlng** [comment]
Hey👋
Thank you for your feedback. I have addressed all the issues/suggestions you mentioned:

1. Resolve Case Sensitivity
wordle 2023-02-10 at 00 16 48

2. Add Delete Summary action
wordle 2023-02-10 at 00 18 16

3. Set `Start Playing` action of selected language as first action
wordle 2023-02-10 at 00 17 45
wordle 2023-02-10 at 00 17 39

Thanks again for taking the time reviewing my extension!

File: extensions/wordle/package.json (hunk 1 above)
Comment (chrisalxlng) [reply extensions/wordle/package.json:15]: added subtitle

File: extensions/wordle/package.json (hunk 2 above)
Comment (chrisalxlng) [reply extensions/wordle/package.json:22]: added subtitle

File: extensions/wordle/package.json (hunk 3 above)
Comment (chrisalxlng) [reply extensions/wordle/package.json:29]: added subtitle

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Looks good to me, approved 🔥

**Pull request merged**